static/images/qiskit-aqua-logo.svg

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
All these SVGs can be cleaned and minified, I'm very good at it. I have created a new issue to solve it in other moment: https://github.com/Qiskit/qiskit.org/issues/315